.changelog/1961.trivial.md

```diff
@@ -0,0 +1 @@
+Restrict hover-based address highlighting to desktop
```

Comment:
why?

Reply:
Beause @donouwens has said so. (I asked him to elaborate on this here.)